src/jsonrpc/miner_jsonrpc_sc.erl

```diff
@@ -7,18 +7,18 @@
 %% jsonrpc_handler
 %%
 handle_rpc(<<"sc_active">>, []) ->
-    case (catch blockchain_state_channels_server:active_sc_ids()) of
-        {'EXIT', _} -> ?jsonrpc_error(timeout);
-        undefined -> #{<<"active">> => []};
-        BinIds -> #{<<"active">> => [ ?TO_VALUE(base64:encode(I)) || I <- BinIds ]}
+    case (catch blockchain_state_channels_server:get_actives()) of
+        {'EXIT', _} ->
```

Comment:
(making sure I understand what's going on here) are we catching this call b/c this is part of the RPC public API and the calling process is trapping exits (hence the first `case` clause) but we're not catching the same call below in `format_sc_map/1` its an internal function?